### ISSUE TITLE
Add user db model

### ISSUE THREAD

**Issue by sils** [coala/confer-backend#7]
@swmerko I guess this is trivial. I think we'll want a user model similar to what we have at https://gitlab.com/gitmate/gitmate-2 and allow a social login using a django plugin (we have that too at gitmate-2) so people can make an account and click "attend" on a conference, later we'll have to add more stuff.

CC @nkprince007 from gitmate who did most of the social auth stuff and @spideralessio for the frontend par (forgot the other username it was so hard to spell ^^)

**Comment by naveenkumarsangi** [comment]
Yeah, the social auth could be used with `python-social-auth[django]` and `python3-openid` in conjunction. They have excellent documentation too.

**Comment by swmerko** [comment]
is it ok the standard implementation in gitmate-2?
